.github/workflows/unittests.yaml

```diff
@@ -0,0 +1,64 @@
+name: unit tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v1
+        with:
+          java-version: openjdk@8
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
Is these needed with more recent versions of https://github.com/olafurpg/setup-scala (it is on v12 now and I didn't see it referenced in docs)